packages/db/src/models/granule.ts

```diff
@@ -15,6 +15,22 @@ export default class GranulePgModel extends BasePgModel<PostgresGranule, Postgre
     });
   }
 
+  /**
+   * Deletes the item from Postgres
+   *
+   * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+   * @param {Partial<RecordType>} params - An object or any portion of an object of type RecordType
+   * @returns {Promise<number>} The number of rows deleted
+   */
+  async delete(
+    knexOrTransaction: Knex | Knex.Transaction,
+    params: PostgresGranuleUniqueColumns | { cumulus_id: number }
```

Comment:
Same thread as the other comment on cumulus_id vs. these unique columns. This feels a little weird to me but I think I'm missing something. 

If we always use the `cumulus_id` we can avoid each model having to overwrite the base `delete` method and the `PostgresGranuleUniqueColumns` type, which seems desirable.

Reply:
There could be a situation where someone wants to a delete without having to lookup the granule first to get the `cumulus_id`. I don't know if we have that in practice now, but we easily could.

Thus, we want this implementation to ensure that if someone is trying to do a granule delete in that scenario, they **have to** pass in the minimal set of fields necessary to uniquely identify a granule so we're sure that they're just deleting one record